thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -1565,10 +1573,9 @@ class CInMemJoinBase : public CSlaveActivity, public CAllOrLookupHelper<HELPER>,
         }
         else
         {
-            CAsyncCallStart asyncLeftStart(std::bind(&CInMemJoinBase::startLeftInput, this));
+            CAsyncCallStart asyncLeftStart(std::bind(&CInMemJoinBase::startLeftInputAsync, this));
```

Comment:
instead of introducing a new method, can pass (bind) the bool here:
```
            CAsyncCallStart asyncLeftStart(std::bind(&CInMemJoinBase::startLeftInput, this, true));
```